src/sentry/apidocs/hooks.py

```diff
@@ -74,12 +75,55 @@ def __get_explicit_endpoints() -> List[Tuple[str, str, str, Any]]:
     ]
 
 
+def __get_line_count_for_team_stats(team_stats: Mapping):
+    line_count = 3
+    for group in team_stats:
+        if len(team_stats[group]) == 0:
+            line_count += 1
+        else:
+            line_count += len(team_stats[group]) + 2
+    return line_count
+
+
+def __write_ownership_data(ownership_data: Mapping):
+    processed_data = {}
+    index = 2
+    for team in ownership_data:
+        processed_data[team.value] = {
+            "index": index,
+            ApiPublishStatus.PUBLIC.value: list(ownership_data[team][ApiPublishStatus.PUBLIC]),
+            ApiPublishStatus.PRIVATE.value: list(ownership_data[team][ApiPublishStatus.PRIVATE]),
+            ApiPublishStatus.EXPERIMENTAL.value: list(
+                ownership_data[team][ApiPublishStatus.EXPERIMENTAL]
+            ),
+            ApiPublishStatus.UNKNOWN.value: list(
+                ownership_data[team][ApiPublishStatus.EXPERIMENTAL.UNKNOWN]
+            ),
+        }
+        index += __get_line_count_for_team_stats(ownership_data[team])
+
+    file_to_write = open(_OWNERSHIP_FILE_PATH, "w")
+    file_to_write.writelines(json.dumps(processed_data, indent=4))
+    file_to_write.close()
+
+
 def custom_preprocessing_hook(endpoints: Any) -> Any:  # TODO: organize method, rename
     filtered = []
+    # data { owner: issues, public: 3, private: 4, unknown: 5, unknown_apis = []}
```

Comment:
what do the numbers signify here? 3,4,5?

Reply:
Will add better comment. They mean this team has 3 public APIs. Mostly important for unknown and experimental which we try to get to 0.